dstat.yaml

```diff
@@ -0,0 +1,46 @@
+package:
+  name: dstat
+  version: 0.7.4
+  epoch: 0
+  description: A versatile resource statistics tool
+  copyright:
+    - license: GPL
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - build-base
+      - python3-dev
+
+pipeline:
+  - uses: fetch
+    with:
+      uri: https://github.com/dagwieers/dstat/archive/v${{package.version}}.tar.gz
+      expected-sha512: d100223887ebd83d0fd1259436f39419a85df9002556abcfc8e4195aa63be73d85707dcd5c4fb56a979b29131485dce6e97c177fbfca08dc50bd0f92b15cc6f5
+
+  - uses: patch
+    with:
+      # The patch source is https://github.com/efexgee/dstat/commit/220a785321b13b6df92a536080aca6ef1cb644ad.patch
+      patches: 220a785321b13b6df92a536080aca6ef1cb644ad.patch
+
+  - runs: |
+      make DESTDIR=${{targets.destdir}} install
+      make DESTDIR="${{targets.destdir}}" docs
+      # Get rid of deprecation warning.
+      sed -i "s/import collections/import collections.abc/g" -i "${{targets.destdir}}"/usr/bin/dstat
+      sed -i "s/collections.Sequence/collections.abc.Sequence/g" -i "${{targets.destdir}}"/usr/bin/dstat
+
+  - uses: strip
+
+subpackages:
+  - name: dstat-doc
+    description: dstat manpages
+    pipeline:
+      - uses: split/manpages
+
+update:
+  enabled: true
+  release-monitor:
+    identifier: 630
```

Comment:
I think we prefer to use the GitHub update functionality over release-monitor when we're getting the source code from GitHub.

But also this project is no longer maintained according to its README

Reply:
I think we should disable the updates as you said this project is not actively maintained any more, thanks.